handlers/restart_consul.yml

```diff
@@ -6,13 +6,13 @@
   when: ansible_service_mgr == "systemd"
   listen: 'reload systemd daemon'
 
-- name: Restart consul on Unix
-  service:
-    name: consul
-    state: restarted
-    # Needed to force SysV service manager on Docker for Molecule tests
-    use: "{{ ansible_service_mgr }}"
+- name: Rolling restart consul on Unix
+  # Manual loop on playbook hosts to perform a "serial: 1" equivalent on a set of task
+  include_tasks: restart-task.yml
```

Comment:
Shouldn't this be `leave_restart_consul.yml` (which should be in `tasks`, not `handlers`)?

Reply:
Indeed! I renamed it and forgot to update the include_tasks. That's fixed.